rfcs/InputUnion.md

```diff
@@ -228,52 +244,19 @@ Preferably without loss of functionality.
 
 The less typing and fewer bytes transmitted, the better.
 
+* Objection: The quantity of "typing" isn't a worthwhile metric, most interactions with an API are programatic.
+* Objection: Simply compressing an HTTP request will reduce the bytes transmitted more than anything having to do with the structure of a Schema.
+
 ### L) Input unions should be performant for servers
 
 Ideally a server does not have to do much computation to determine which concrete type is represented by an input.
 
-## Use Cases
-
-There have been a variety of use cases described by users asking for an abstract input type.
-
-* [Observability Metrics](https://github.com/graphql/graphql-spec/pull/395#issuecomment-489495267)
-* [Login Options](https://github.com/graphql/graphql-js/issues/207#issuecomment-228543259)
-* [Abstract Syntax Tree](https://github.com/graphql/graphql-spec/pull/395#issuecomment-489611199)
-* [Content Widgets](https://github.com/graphql/graphql-js/issues/207#issuecomment-308344371)
-* [Filtering](https://github.com/graphql/graphql-spec/issues/202#issue-170560819)
-* [Observability Cloud Integrations](https://gist.github.com/binaryseed/f2dd63d1a1406124be70c17e2e796891#cloud-integrations)
-* [Observability Dashboards](https://gist.github.com/binaryseed/f2dd63d1a1406124be70c17e2e796891#dashboards)
+* Objection: None of the solutions discussed so far do anything that is computationally expensive.
 
 
 ## Possible Solutions
 
-Each of the solutions should be evaluated against each of the criteria.
```

Comment:
I think this overview is the most useful tool for us to eventually come to a conclusion. Let's not move backwards.